tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_joint
    joint_speed: 1.0
    motor:
        id: 3
        init: 512
        min: 511
        max: 820

pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: pan_joint
    joint_speed: 1.0
    motor:
        id: 2
        init: 512
        min: 0
        max: 1023
